MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1764,10 +1765,10 @@ public bool CanStreamCopyVideo(EncodingJobInfo state, MediaStream videoStream)
             {
                 if (string.IsNullOrEmpty(videoStream.VideoRangeType))
                 {
-                    return false;
+                    // return false;
```

Comment:
The default value of VideoRangeType is SDR. This can’t be missing.
And actually remuxing/direct streaming works fine for me in both SDR and HDR/DV content.
What are you trying to fix here?
https://github.com/jellyfin/jellyfin/blob/84a1674f391d344c251c2c949cb227a9318536eb/MediaBrowser.Model/Entities/MediaStream.cs#L692

Reply:
@nyanmisaka ah yes, the actual issues is the copy&paste error for the check!
Updated the PR

The issues is that if you send a stream request and have no "rangetype" this always triggers a transcode.